app/style/MiscComponents.less

```diff
@@ -647,7 +652,15 @@
 }
 
 .no-transactions-indicator {
+  color: #617386;
```

Comment:
Where did this color come from?  You need to use the design specs to choose the proper color.

Reply:
That is from the specs, it is the color of the text.
https://xd.adobe.com/spec/5a5726c5-e6ea-4273-6262-82582819feec-2c66/screen/f18f4868-bc65-4642-ae12-a7611037d606/Tickets-My-Tickets-Empty-state-3/

Reply:
Gotcha, just didn't recognize it.